app/views/PremiumFeaturesView.coffee

```diff
@@ -20,9 +21,9 @@ module.exports = class PremiumFeaturesView extends RootView
     super(options)
     
   afterInsert: () ->
-    super()
-    if utils.getQueryVariable('pop')?
+    if utils.getQueryVariable('pop')? and not storage.load('sub-modal-continue')
```

Comment:
I'm not sure why this logic is set up this way. Can you explain what this line change is for?

Reply:
This query variable opens a subscribe modal automatically, so this check avoids opening two in this scenario.